.eslintrc

```diff
@@ -6,9 +6,10 @@
     "plugin:vue/essential"
   ],
   "globals": {
-    "Vue": true
+    "Vue": true,
+    "Highcharts": true
```

Comment:
You can make `$` global too so jQuery doesn't have to be imported in other files

Reply:
Yes, we are using it as global now.